IPython/core/tests/test_oinspect.py

```diff
@@ -340,13 +340,13 @@ def test_calldef_none():
 @skipif(not py3compat.PY3)
 def test_definition_kwonlyargs():
     i = inspector.info(f_kwarg, oname='f_kwarg')  # analysis:ignore
-    nt.assert_equal(i['definition'], "f_kwarg(pos, *, kwonly)\n")
+    nt.assert_equal(i['definition'], "f_kwarg(pos, *, kwonly)")
```

Comment:
So I removed the `\n` in the end of definition because it is part of the formatting to me.